media/html/visjsDebuggerPanel.html

```diff
@@ -324,6 +361,49 @@
       function openAllClusters() {
         openClusterStack(Object.keys(network.body.nodes).filter((nodeId) => nodeId.startsWith('cluster_')));
       }
+
+      function showAllNodes() {
+        let nodeIds = Object.keys(network.body.nodes).filter((nodeId) => {
+          return !nodeId.startsWith('edge');
+        });
```

Comment:
```suggestion
        const nodeIds = network ? Object.keys(network.body.nodes).filter((nodeId) => !nodeId.startsWith('edge')) : [];
```
The button is already interactable before the network is created.